src/OXRS_WT32.cpp

```diff
@@ -579,6 +587,15 @@ void OXRS_WT32::_initialiseRestApi(void)
   _server.begin();
 }
 
+void OXRS_WT32::_initialiseESP32TempSensor(void)
```

Comment:
Could we move this inside `initialiseClimateSensor()` so we just have the two `xxxClimateSensor()` methods for handling the sensor setup/readings?